.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+nomad-spark PR Summary
+==============================
+
+Briefly describe how this Pull Request changes the project.
+
+Additional Details
```

Comment:
Generally developers ignore this section. Should we be very specific to add sections for
- How the change was tested ?

Reply:
I don't think this is necessary, as the checklist includes a section for testing.